themes/theme-b2c-tailwind/src/molecules/aggregate-rating.css

```diff
@@ -0,0 +1,24 @@
+[data-store-aggregate-rating] {
+  @apply flex;
+}
+
+[data-store-aggregate-rating] svg[data-store-aggregate-rating-item] {
+  @apply w-6 mr-2;
+}
+
+li[data-store-aggregate-rating-item], span[data-store-aggregate-rating-item] {
+  color: #ffb100;
+}
+
+/* Using CSS to insert icons */
+
+[data-store-aggregate-rating] span[data-store-aggregate-rating-item="full"]:after,
+[data-store-aggregate-rating] span[data-store-aggregate-rating-item="half"]:after {
```

Comment:
Noticed a star went missing when I tested the [Aggregate Rating With CSS](https://deploy-preview-1037--faststoreui.netlify.app/?path=/story/molecules-aggregaterating--aggregate-rating-with-css) example with 3.5:

![Screen Shot 2021-12-07 at 11 11 29](https://user-images.githubusercontent.com/381395/145044574-5ca9cb14-0525-4912-9692-5c65e819b3de.png)

Figured we forgot to rename `half` to `partial` here:

```suggestion
[data-store-aggregate-rating] span[data-store-aggregate-rating-item="partial"]:after {
```

---

After I did that change, I noticed I got 4 full-stars instead of 3:
![Screen Shot 2021-12-07 at 11 13 13](https://user-images.githubusercontent.com/381395/145044879-784d9766-65ac-4590-a70c-6f06cb4261c6.png)

So I think we also need to move `[data-store-aggregate-rating] span[data-store-aggregate-rating-item="partial"]:after` together with `[data-store-aggregate-rating] span[data-store-aggregate-rating-item="empty"]:after`, not the `full` one.

Reply:
It makes sense, gonna fix this one. Really thanks @filipewl!